services/src/main/java/org/apache/druid/cli/CliRouter.java

```diff
@@ -112,7 +114,8 @@ protected List<? extends Module> getModules()
           binder.bind(new TypeLiteral<List<TieredBrokerSelectorStrategy>>() {})
                 .toProvider(TieredBrokerSelectorStrategiesProvider.class)
                 .in(LazySingleton.class);
-
+          
+          binder.bind(JoinableFactory.class).to(NoopJoinableFactory.class).in(LazySingleton.class);
```

Comment:
This shouldn't be necessary anymore, right?

Reply:
removed